ts/IterableInboxMessageList.tsx

```diff
@@ -9,33 +9,33 @@ import InboxRowViewModel from './InboxRowViewModel'
 import Customization from './customizationType'
 
 type MessageListProps = {
-   messages: InboxRowViewModel[],
+   rowViewModels: InboxRowViewModel[],
+   customizations: Customization,
+   deleteMessage: Function,
    messageListItemLayout: Function,
-   customizations: Customization
-   //deleteMessage: Function,
    handleMessageSelect: Function 
 }
 
 const IterableInboxMessageList = ({ 
-   messages,
-   messageListItemLayout,
+   rowViewModels,
    customizations, 
-   //deleteMessage, 
+   deleteMessage, 
+   messageListItemLayout,
    handleMessageSelect 
 }: MessageListProps) => {
    const [swiping, setSwiping] = useState(false)
 
-   const renderMessageCells = (messages: InboxRowViewModel[]) => {
-      return messages.map((message, index) => {
+   const renderMessageCells = (rowViewModels: InboxRowViewModel[]) => {
+      return rowViewModels.map((rowViewModel, index) => {
          return (
             <IterableInboxSwipeableRow
-               key={message.inAppMessage.messageId}
+               key={rowViewModel.inAppMessage.messageId}
                index={index}
-               message={message}
+               rowViewModel={rowViewModel}
                messageListItemLayout={messageListItemLayout}
                customizations={customizations}
                //swipingCheck={(swiping : boolean) => setSwiping(swiping)}
-               //deleteMessage={(id: string) => deleteMessage(id)}
+               deleteMessage={(id: string, index: number) => deleteMessage(id, index)}
```

Comment:
We shouldn't need both the `messageId` and `index` here for deletion.

And we should also clarify what `id` it is.

Reply:
Agreed. I will clarify that it is the message id.